syntax = "proto3";

import "google/protobuf/any.proto";

message Request {
  string func_name = 1;
  repeated google.protobuf.Any args = 2;
}

message Response {
  google.protobuf.Any res = 1;
}

service CallFuncService {
  rpc CallFunc(Request) returns (Response);
}
